recipe/post-link.sh

```diff
@@ -9,13 +9,18 @@
 # available as `$PREFIX/bin/.gdk-pixbuf-post-link.sh`.
 
 set -e
-eval $(grep -v : $PREFIX/lib/pkgconfig/gdk-pixbuf-2.0.pc)
+eval $(grep -v : "$PREFIX/lib/pkgconfig/gdk-pixbuf-2.0.pc")
 if [ -z "$gdk_pixbuf_cache_file" ] ; then
-    bindir="$(echo $PREFIX/lib/gdk-pixbuf-2.0/*)"
-    if [ ! -d $bindir ] ; then
+    bindir="$(echo "${PREFIX}/lib/gdk-pixbuf-2.0/"*)"
+    if [ ! -d "$bindir" ] ; then
        echo >&2 "error: no such gdk-pixbuf binary directory $PREFIX/lib/gdk-pixbuf-2.0/*"
        exit 1
     fi
     gdk_pixbuf_cache_file="$bindir/loaders.cache"
 fi
-"$PREFIX/bin/gdk-pixbuf-query-loaders" >$gdk_pixbuf_cache_file
+"$PREFIX/bin/gdk-pixbuf-query-loaders" >"$gdk_pixbuf_cache_file" 2>>"${PREFIX}/.messages.txt" || \
+(
+    echo "ERROR: Failed to update gdk-pixbuf's cache, some plugins may be be found."
```

Comment:
Confusing typo: "be be".

Please also add a source code comment mentioning that one situation in which the command might fail is if the package is being installed into an environment for a foreign architecture (though I'm not sure if my terminology is fully right for this).